dspace/config/registries/eperson-types.xml

```diff
@@ -38,4 +38,17 @@
         <scope_note>Metadata field used for the language</scope_note>
     </dc-type>
 
+    <dc-type>
+        <schema>eperson</schema>
+        <element>orcid</element>
+        <scope_note>Metadata field used for the ORCID id</scope_note>
+    </dc-type>
```

Comment:
Silly question maybe, but why not just use the existing `person.orcid` on the EPerson object instead of creating a new field?

Reply:
@LucaGiamminonni : Ok, we can keep this around.  But, I still feel that the `eperson.orcid.scope` is unnecessary (see below) since we have a generic `dspace.orcid.scope` field already which meets this need and is not object type specific.